config/fcos/v1_3_exp/translate_test.go

```diff
@@ -0,0 +1,1153 @@
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "raid", 1, "level")},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "raid", 1, "name")},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "raid", 1, "options", 0)},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "raid", 2, "devices", 0)},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "raid", 2, "devices", 1)},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "raid", 2, "level")},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "raid", 2, "name")},
+				{path.New("yaml", "boot_device", "luks", "tang", 0, "url"), path.New("json", "storage", "luks", 0, "clevis", "tang", 0, "url")},
+				{path.New("yaml", "boot_device", "luks", "tang", 0, "thumbprint"), path.New("json", "storage", "luks", 0, "clevis", "tang", 0, "thumbprint")},
+				{path.New("yaml", "boot_device", "luks", "threshold"), path.New("json", "storage", "luks", 0, "clevis", "threshold")},
+				{path.New("yaml", "boot_device", "luks", "tpm2"), path.New("json", "storage", "luks", 0, "clevis", "tpm2")},
+				{path.New("yaml", "boot_device", "luks"), path.New("json", "storage", "luks", 0, "device")},
+				{path.New("yaml", "boot_device", "luks"), path.New("json", "storage", "luks", 0, "label")},
+				{path.New("yaml", "boot_device", "luks"), path.New("json", "storage", "luks", 0, "name")},
+				{path.New("yaml", "boot_device", "luks"), path.New("json", "storage", "luks", 0, "wipeVolume")},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "filesystems", 0, "device")},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "filesystems", 0, "format")},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "filesystems", 0, "label")},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "filesystems", 0, "wipeFilesystem")},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "filesystems", 1, "device")},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "filesystems", 1, "format")},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "filesystems", 1, "label")},
+				{path.New("yaml", "boot_device", "mirror"), path.New("json", "storage", "filesystems", 1, "wipeFilesystem")},
+				{path.New("yaml", "boot_device"), path.New("json", "storage", "filesystems", 2, "device")},
+				{path.New("yaml", "boot_device"), path.New("json", "storage", "filesystems", 2, "format")},
+				{path.New("yaml", "boot_device"), path.New("json", "storage", "filesystems", 2, "label")},
+				{path.New("yaml", "boot_device"), path.New("json", "storage", "filesystems", 2, "wipeFilesystem")},
+			},
+		},
+		// 2-disk mirror + LUKS with overridden root partition size
```

Comment:
On the subject of overridding the root partition size and just general overrides, are we missing docs somewhere about what e.g. labels to use to make sure it matches up with what the sugar outputs?

Reply:
Yeah, we need docs for this feature generally.  Let's defer that to a followup?